### ISSUE TITLE
[BUG] meta + another key will cause keyup callback to never fire

### ISSUE THREAD

**Issue by HaydenJH** [JohannesKlauss/react-hotkeys-hook#1018]
**Describe the bug**
When binding to meta key + any other key and listening for keydown and keyup events the callback is never fired with a keyup event

**To Reproduce**
1. Go to  [example](https://codesandbox.io/s/react-hotkeys-hook-issue-1013-forked-exp648)
2. Press Ctrl b and release, observe it bolds then unbolds
3. Press meta b and release, observe it stays bold

Please try and add a codesandbox or stackblitz to reproduce the bug:
[codesandbox](https://codesandbox.io/s/react-hotkeys-hook-issue-1013-forked-exp648)

**Expected behavior**
The keydown should fire on keydown and the keyup should fire on 

**Desktop (please complete the following information):**
 - OS: macOS Monterey 12.6
 - Browser: Chrome
 - Version: 112.0.5615.137 (

**Comment by JohannesKlauss** [comment]
Yes, this is sadly a behavior in macOS and there is no fix for it. macOS just does not propagate a keyup event as long as the meta key is pressed down.

You can check this behavior here: https://codesandbox.io/s/affectionate-wright-hr5ksz?file=/src/index.js

As soon as you hold down the meta key, no keyup event will be triggered, no matter what key you press.

There is nothing we can do about that.

**Comment by enjoijkee** [comment]
though 4 years old [react-hotkeys](https://www.npmjs.com/package/react-hotkeys) handles this behaviour pretty well...
`{ sequence: 'cmd+right', action: 'keyup' }`
Just switched from it to the react-hotkeys-hook for another reason and got stuck at this issue.

**Comment by JohannesKlauss** [comment]
I'll have a look at it how they are handling this issue

**Comment by sebak94** [comment]
Any news?

**Comment by JohannesKlauss** [comment]
You're welcome to contribute 🙂 I have a day job, so my time working on this library is very limited.

**Comment by bryanmylee** [comment]
Maybe re-open the issue until this is fixed?

**Comment by mikhael28** [comment]
As an fyi, you can attach a Mac OS cmd shortcut without any library:
 useEffect(() => {
    const handleKeyDown = (e: KeyboardEvent) => {
      // Check for both Ctrl and Command (Meta) keys
      if ((e.ctrlKey || e.metaKey) && e.key === "k") {
        e.preventDefault();
        setIsOpen((prev) => !prev); // Toggle the palette
      } else if (e.key === "Escape") {
        setIsOpen(false);
      }
    };

    window.addEventListener("keydown", handleKeyDown);
    return () => window.removeEventListener("keydown", handleKeyDown);
  }, []);

You have to check that the 'meta' key is pressed down.
